clients/rancher/client.go

```diff
@@ -113,6 +115,14 @@ func NewClient(bearerToken string, session *session.Session) (*Client, error) {
 
 	c.WranglerContext = wranglerContext
 
+	splitBearerKey := strings.Split(bearerToken, ":")
+	token, err := c.Management.Token.ByID(splitBearerKey[0])
+	if err != nil {
+		return nil, err
+	}
+
+	c.UserID = token.UserID
+
 	return c, nil
 }
 
```

Comment:
Unless i'm greatly mistaken these changes were from this [PR](https://github.com/rancher/shepherd/pull/250) and have already been merged.